game/scripts/npc/abilities/slardar_bash_oaa.txt

```diff
@@ -22,7 +22,7 @@
 
     // Time
     //-------------------------------------------------------------------------------------------------------------
-    "AbilityCooldown"                                     "4.0"
+    "AbilityCooldown"                                     "4.4"
```

Comment:
lmao, 4 or 5 seconds is fine, putting decimals is weird

Reply:
wow ignorin me